Item "type" of "Union[Type[Foo], Type[Bar]]" has no attribute "x"

```
# mypy 0.530
foo.py:14: warning: Returning Any from function declared to return "int"
foo.py:14: error: Item "type" of "Union[Type[Foo], Type[Bar]]" has no attribute "x"
```

```python
from typing import *

class Foo(NamedTuple):
    a: int
    @classmethod
    def x(cls) -> int: return 1

class Bar(NamedTuple):
    a: int
    @classmethod
    def x(cls) -> int: return 1

def foo(t: Type[Union[Foo, Bar]]) -> int:
    return t.x()
```

If I make them usual classes instead of namedtuples, it works

--- a/mypy/checkmember.py
+++ b/mypy/checkmember.py
@@ -191,6 +191,8 @@ def analyze_member_access(name: str,
         elif isinstance(typ.item, TypeVarType):
             if isinstance(typ.item.upper_bound, Instance):
                 item = typ.item.upper_bound
+        elif isinstance(typ.item, TupleType):
+            item = typ.item.fallback
         elif isinstance(typ.item, FunctionLike) and typ.item.is_type_obj():
             item = typ.item.fallback
         elif isinstance(typ.item, TypeType):

--- a/test-data/unit/check-namedtuple.test
+++ b/test-data/unit/check-namedtuple.test
@@ -638,3 +638,14 @@ class A(NamedTuple):
 
 class B:
     pass
+
+[case testTypeNamedTupleClassmethod]
+from typing import Type, NamedTuple
+class D(NamedTuple):
+    @classmethod
+    def f(cls) -> None: pass
+
+d: Type[D]
+d.g()  # E: "Type[D]" has no attribute "g"
+d.f()
+[builtins fixtures/classmethod.pyi]

--- a/test-data/unit/check-tuples.test
+++ b/test-data/unit/check-tuples.test
@@ -940,3 +940,15 @@ y = ""
 reveal_type(t[x:])  # E: Revealed type is 'Union[builtins.int, builtins.str]'
 t[y:]  # E: Slice index must be an integer or None
 [builtins fixtures/tuple.pyi]
+
+[case testTypeTupleClassmethod]
+from typing import Tuple, Type
+
+class C(Tuple[int, str]):
+    @classmethod
+    def f(cls) -> None: pass
+
+t: Type[C]
+t.g()  # E: "Type[C]" has no attribute "g"
+t.f()
+[builtins fixtures/classmethod.pyi]

